pkg/sql/logictest/tests/local-vec-off/local-vec-off_test: TestLogic_system failed

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_system [failed](https://tanzanite.cluster.engflow.com/invocations/default/8fbe1ff7-7f18-4b42-860d-6d43bd03d00a?testReportRun=7&testReportShard=45&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9sb2NhbC12ZWMtb2ZmOmxvY2FsLXZlYy1vZmZfdGVzdA==) on release-24.3 @ [047a8e99eb7a35e3f13ecf6b113f1965f206e3b9](https://github.com/cockroachdb/cockroach/commits/047a8e99eb7a35e3f13ecf6b113f1965f206e3b9):


```
=== RUN   TestLogic_system
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestLogic_system2637775106
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    logictestbase.go:550: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports 3node-tenant-default-configs (no issue given)
    logictestbase.go:550: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports 3node-tenant-default-configs (no issue given)
    logictestbase.go:550: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports 3node-tenant-default-configs (no issue given)
    logictestbase.go:550: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports 3node-tenant-default-configs (no issue given)
    logictestbase.go:550: statement/query skipped with reason: unsupported configuration local-vec-off, statement/query only supports 3node-tenant-default-configs (no issue given)
    logic.go:3043: 
         
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/system:1443: SHOW CLUSTER SETTING sql.defaults.vectorize
        expected:
            on
            
        but found (query options: "") :
            off
            
[06:06:59] --- done: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/system with config local-vec-off: 75 tests, 1 failures
    logic.go:4229: 
        /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/system:1448: error while processing
    logic.go:4229: /var/lib/engflow/worker/work/1/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/system:1448: too many errors encountered, skipping the rest of the input
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_system2637775106
--- FAIL: TestLogic_system (1.39s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=7</code>
 - <code>shard=45</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_system.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-43602

--- a/pkg/sql/logictest/testdata/logic_test/system
+++ b/pkg/sql/logictest/testdata/logic_test/system
@@ -1440,7 +1440,7 @@ SELECT name, value FROM system.settings WHERE name = 'sql.defaults.vectorize'
 ----
 sql.defaults.vectorize  1
 
-query T
+query T retry
 SHOW CLUSTER SETTING sql.defaults.vectorize
 ----
 on
@@ -1468,7 +1468,7 @@ query TT
 SELECT name, value FROM system.settings WHERE name = 'sql.defaults.vectorize'
 ----
 
-query T
+query T retry
 SHOW CLUSTER SETTING sql.defaults.vectorize
 ----
 on

